lib/signs/utilities/headways.ex

```diff
@@ -58,6 +80,15 @@ defmodule Signs.Utilities.Headways do
       }}}
   end
 
+  defp get_paging_headway_message(config, destination, headways) do
+    {config,
```

Comment:
Hmm good catch! Looked into this a bit and you're correct in that it's not returning the right type given the status quo, but I think it technically doesn't break the typespec. The typespec for `Utilities.Messages.get_messages` says that it should return a type `sign_messages` which is defines the first element of a given message tuple as `Signs.Utilities.SourceConfig.config()` which is defined [here](https://github.com/mbta/realtime_signs/blob/main/lib/signs/utilities/source_config.ex#L136) as either a tuple of a single list, or a tuple of two lists. As a matter of fact, this is weird because the usual messages return a `SourceConfig.source()` as the type, which _doesn't_ match the return type of`sign_messages` in the typespec 🤔. FWIW, I will say that I think Elixir's typespecs and more generally that Elixir is dynamically typed, are some of the more frustrating/annoying parts of Elixir.

Also, we can't remove the configs completely though because the source configs do get referenced in the audio utilities when determining if we need to send an interrupting read for example [here](https://github.com/mbta/realtime_signs/blob/main/lib/signs/utilities/audio.ex#L280).

Reply:
This comment made me realize though that we're not handling these new message types very well in that `signs/utilities/audio.ex` file which should be addressed